lib/BirthdayCalendarGenerator.php

```diff
@@ -142,9 +141,10 @@ public function getResult()
             }
 
             // Create event.
+            /** @var VEvent $event */
```

Comment:
Do we really need this? Or shouldn't a proper return type for the called method be sufficient?

Reply:
I am looking through the diffs at the moment, and quite a few things like this will disappear.
cs-fixer finds and removes a lot of them.

Reply:
It is not needed. phpstan does not complain without it, so it is not a helper for code-analysis. I removed it.